recipes/benchmarks/fmbench/README.md

```diff
@@ -0,0 +1,117 @@
+
+![Effect of prompt size on inference latency for different concurrency levels](./img/latency_vs_tokens.png)
+
+### How to get started with `FMBench`
+
+The following steps provide a Quick start guide for `FMBench`. For a more detailed DIY version, please see the [`FMBench Readme`](https://github.com/aws-samples/foundation-model-benchmarking-tool?tab=readme-ov-file#the-diy-version-with-gory-details).
+
+1. Launch the AWS CloudFormation template included in this repository using one of the buttons from the table below. The CloudFormation template creates the following resources within your AWS account: Amazon S3 buckets, Amazon IAM role and an Amazon SageMaker Notebook with this repository cloned. A read S3 bucket is created which contains all the files (configuration files, datasets) required to run `FMBench` and a write S3 bucket is created which will hold the metrics and reports generated by `FMBench`. The CloudFormation stack takes about 5-minutes to create.
+
+   |AWS Region                |     Link        |
+   |:------------------------:|:-----------:|
+   |us-east-1 (N. Virginia)    | [<img src="./img/CFT.png">](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=fmbench&templateURL=https://aws-blogs-artifacts-public.s3.amazonaws.com/artifacts/ML-FMBT/template.yml) |
+   |us-west-2 (Oregon)    | [<img src="./img/CFT.png">](https://console.aws.amazon.com/cloudformation/home?region=us-west-2#/stacks/new?stackName=fmbench&templateURL=https://aws-blogs-artifacts-public.s3.amazonaws.com/artifacts/ML-FMBT/template.yml) |
+
+1. Once the CloudFormation stack is created, navigate to SageMaker Notebooks and open the `fmbench-notebook`.
+
+1. On the `fmbench-notebook` open a Terminal and run the following commands.
+
+    ```{.bash}
+    conda create --name fmbench_python311 -y python=3.11 ipykernel
+    source activate fmbench_python311;
+    pip install -U fmbench
+    ```
+
+1. Now you are ready to `fmbench` with the following command line. We will use a sample config file placed in the S3 bucket by the CloudFormation stack for a quick first run.
+
+    1. We benchmark performance for the `Llama2-7b` model on a `ml.g5.xlarge` and a `ml.g5.2xlarge` instance type, using the `huggingface-pytorch-tgi-inference` inference container. This test would take about 30 minutes to complete and cost about $0.20.
+
+    1. It uses a simple relationship of 750 words equals 1000 tokens, to get a more accurate representation of token counts use the `Llama2 tokenizer` (instructions are provided in the next section). ***It is strongly recommended that for more accurate results on token throughput you use a tokenizer specific to the model you are testing rather than the default tokenizer. See instructions provided later in this document on how to use a custom tokenizer***.
```

Comment:
can we clarify this part a bit, are we packing the requests at 1k tokens? how batching works here? is it packing instead of batching?

Reply:
> See instructions provided later in this document on how to use a custom

Shall we pls add a link to where we are providing the instructions?

Reply:
> can we clarify this part a bit, are we packing the requests at 1k tokens? how batching works here? is it packing instead of batching?

No we are not limiting the requests at 1k tokens, what we are saying here is that if you do not provide a custom tokenizer for the model under test then we use the relation 750 words is 1000 tokens to count tokens (token counting is needed to filter prompts if we want to limit prompts within a given size limit, and also for metrics calculation).

Reply:
> instructions provided later in this document on how to use a custom

Thank you for pointing this out, will rephrase this to point to the `Tokenizer Directory` section [here](https://github.com/aws-samples/foundation-model-benchmarking-tool/tree/main?tab=readme-ov-file#the-diy-version-with-gory-details).